doc/theoretical_description_classification.rst

```diff
@@ -84,28 +105,49 @@ However, its incorporation can also be chosen randomly based on the difference b
 coverage remains close to the target (marginal) coverage. We refer the reader to [2, 3] for more details about this aspect.
 
 
-3. Top-K
---------
 
-Introduced by [3], the specificity of the Top-K method is that it will give the same prediction set size for all observations.
-The conformity score is the rank of the true label, with scores ranked from higher to lower.
-The prediction sets are build by taking the :math:`\hat{q}^{th}` higher scores. The procedure is described in the following equations : 
+4. Regularized Adaptive Prediction Sets (RAPS)
+----------------------------------------------
+
+The RAPS method which stands for Regularized Adaptive Prediction Set, is an improvement made by Angelopoulos et al. in 
+[3]. This regularization is able to overcome the very large prediction sets given by the APS method. The conformity scores are
+computed by summing the regularized ranked scores of each label, from the higher to the lower until reaching the true label of the observation :
 
 .. math:: 
-   s_i(X_i, Y_i) = j \quad \text{where} \quad Y_i = \pi_j \quad \text{and} \quad \hat{\mu}(X_i)_{\pi_1} > ... > \hat{\mu}(X_i)_{\pi_j} > ... > \hat{\mu}(X_i)_{\pi_n}
+   s_i(X_i, Y_i) = \sum^k_{j=1} \hat{\mu}(X_i)_{\pi_j} + \lambda (k-k_{reg})^+ \quad \text{where} \quad Y_i = \pi_k
```

Comment:
I don't understand \text{where} \quad Y_i = \pi_k

Reply:
what don't you understand ?

Reply:
What does mean Y_i = \pi_k. pi_k is an index and Y_i a label, no?